src/main/java/org/cookieandkakao/babting/domain/food/controller/FoodPreferenceController.java

```diff
@@ -38,22 +41,26 @@ public FoodPreferenceController(PreferenceFoodService preferenceFoodService,
 
     // 선호/비선호 음식 조회
     @GetMapping("/{type}")
-    public ResponseEntity<?> getFoodPreferences(
+    @Operation(summary = "선호/비선호 음식 조회", description = "내 선호/비선호 음식을 가져옵니다.")
+    @ApiResponse(responseCode = "200", description = "선호/비선호 음식 조회 성공")
+    public ResponseEntity<ApiResponseBody.SuccessBody<List<FoodPreferenceGetResponse>>> getFoodPreferences(
             @PathVariable String type,
             @LoginMemberId Long memberId
     ) {
         FoodPreferenceStrategy strategy = getStrategy(type);
         List<FoodPreferenceGetResponse> preferences = strategy.getAllPreferencesByMember(memberId);
 
         if (preferences.isEmpty()) {
-            return ApiResponseGenerator.fail(HttpStatus.OK, "조회된 음식이 없습니다");
+            return ApiResponseGenerator.success(HttpStatus.OK, "조회된 음식이 없습니다", null);
```

Comment:
```suggestion
            return ApiResponseGenerator.success(HttpStatus.OK, "조회된 음식이 없습니다");
```
ApiResponseGenrator의 success함수가 오버로딩 돼있어서 data가 null일때는 굳이 안써줘도 될 것 같아요.

Reply:
메서드의 반환 타입이 ResponseEntity<SuccessBody<List<FoodPreferenceGetResponse>>>라서  null을 사용하지 않으면 오류가 발생합니다ㅠㅠ
사실 ResponseEntity<?>를 사용하면 가능하긴 한데 멘토님께서 와일드카드 사용은 지양하라고 하셔서 불가피하게 null을 붙였습니다

Reply:
혹시 그냥 빈 리스트인 preferences를 보내주는 건 별롤까요?

Reply:
그것도 괜찮을 것 같네요! 일단 그렇게 수정하겠습니다